jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/Utils.kt

```diff
@@ -285,3 +294,59 @@ fun isCodeModernizerAvailable(project: Project): Boolean {
 }
 
 fun isGradleProject(project: Project) = !GradleSettings.getInstance(project).linkedProjectsSettings.isEmpty()
+
+object TransformUtils {
+    private val LOG = getLogger<TransformUtils>()
+
+    fun getJavaVersionFromProjectSetting(project: Project): String? = project.tryGetJdk()?.toString()
```

Comment:
Nit. Doesn't need access to logger.

Reply:
you can also do `LoggerFactory.getLogger("...")` if it's a util-only file and you need a logger that isn't tied to a class/object

Reply:
I see, that is much better!